Bindings/android/bindings/build.gradle

```diff
@@ -41,10 +41,10 @@ dependencies {
     implementation 'com.google.android.material:material:1.3.0'
 
     embed "com.tealium:remotecommands:1.0.1"
-    embed "com.tealium:kotlin-core:1.4.1"
+    embed "com.tealium:kotlin-core:1.5.1"
     embed "com.tealium:kotlin-collect-dispatcher:1.1.0"
     embed "com.tealium:kotlin-lifecycle:1.1.1"
-    embed "com.tealium:kotlin-remotecommand-dispatcher:1.1.1"
-    embed "com.tealium:kotlin-tagmanagement-dispatcher:1.1.0"
-    embed "com.tealium:kotlin-visitor-service:1.1.0"
+    embed "com.tealium:kotlin-remotecommand-dispatcher:1.2.0"
```

Comment:
if it's a straigthforward switch, then `1.2.1` is the latest version of the RemoteCommandDispatcher